plugin/src/main/java/org/jenkinsci/plugins/structs/SymbolLookup.java

```diff
@@ -152,5 +153,29 @@ public static SymbolLookup get() {
         return j.getInjector().getInstance(SymbolLookup.class);
     }
 
+    /**
+     * Get the {@link Symbol} value for the class of the given object, if the annotation is present
+     *
+     * @param o An object
```

Comment:
This is going to be rather confusing, because `o` should in most cases be a `Descriptor` not a `Describable`.